tests/parsing_test.py

```diff
@@ -0,0 +1,79 @@
+import tempfile
+import unittest
+
+from Bio import Entrez
+
+from metab_classes import Publication
+from metab_pub_ingest import Citation
+from metab_pub_ingest import fill_pub
+
+
+class TestParse(unittest.TestCase):
+    def test_parse(self):
+        """ Confirm citations are being written in the correct format """
+        results = self.create_citation()
+        citation = Citation(results['PubmedArticle'][0])
+        pub = Publication()
+        fill_pub(pub, citation)
+
+        expected_citation = "Smith, J., Doe, J., Hamill, M. (2007). This is an example publication. Journal Of Example Science, 1(5), 100-105. doi:12.3456/7890"
+        self.assertEqual(pub.pmid, '999999')
+        self.assertEqual(pub.citation, expected_citation)
+
+    def create_citation(self):
+        with tempfile.TemporaryFile() as handle:
```

Comment:
Can you use [io.StringIO](https://docs.python.org/3/library/io.html) instead of `tempfile`?